spec/factories/courses.rb

```diff
@@ -52,6 +52,16 @@
       )
     }
 
+    trait :primary do
+      level { :primary }
+      age_range_in_years { "3_to_7" }
+    end
+
+    trait :secondary do
+      age_range_in_years { "16_to_18" }
```

Comment:
Minor: Can we do 11 to 18, as I think that's most common

Reply:
16 to 18 is an invalid age, courses must span an age range of more than 4 years.

Reply:
![Screen Shot 2019-10-25 at 15 42 46](https://user-images.githubusercontent.com/319055/67580370-20b0c700-f73e-11e9-8411-eeb337e666b5.png)

Reply:
Yes of course. Thanks for pointing this out @fofr

Reply:
feeling like we going to have a whack mole moment....
https://github.com/DFE-Digital/manage-courses-backend/pull/925